Added accessibility menu traversal with keyboard

sergiogcx opened the pull request.

briaguya commented:
just played around with the PR app and it's looking really good!

Couple of little notes:
Right now we need to tab through all of the headers before we can get to the links inside, and all of the headers just open the menu. Not sure how we want the keyboard UX for this to work.

I know we discussed a "click off" option for this instead of just clicking the "X," not sure where we landed with it (it can probably be it's own issue if we decide to move forward with it)

sergiogcx commented:
@briaguya Yes, so my bad I had not initially understood the click-off part of it. I added it shortly after your comment. I also just added going through the theme headings in the menu, before going through each link under it.

easherma commented:
One other thing I noticed from the video, do we need to add aria labels for
the topic headings themselves? The screen-reader didn't read off what the
headings were, just the links inside them.

On Fri, May 10, 2019 at 2:29 AM Sergio Garcia <notifications@github.com>
wrote:

> @briaguya <https://github.com/briaguya> Yes, so my bad I had not
> initially understood the click-off part of it. I added it shortly after
> your comment. I also just added going through the theme headings in the
> menu, before going through each link under it.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/cityofaustin/janis/pull/397#issuecomment-491188445>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACQNGT2WAFOYDK2L665JSYTPUUP4VANCNFSM4HL7YPWA>
> .
>


-- 
Eric Sherman
224-402-0004
